# 1. remove ".pre2" from version
#    configure.ac and configure still defined version "5.9.4.pre2" instead of "5.9.4"
# 2. remove cross_compile check to use sysctl
#    allow NETSNMP_CAN_USE_SYSCTL with ac_cv_NETSNMP_CAN_USE_SYSCTL=yes
# 
--- configure.orig	2023-08-15 20:32:24.000000000 +0000
+++ configure	2024-06-22 08:21:11.991222691 +0000
@@ -1,7 +1,7 @@
 #! /bin/sh
 # From configure.ac Revision.
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.71 for Net-SNMP 5.9.4.pre2.
+# Generated by GNU Autoconf 2.71 for Net-SNMP 5.9.4.
 #
 # Report bugs to <net-snmp-coders@lists.sourceforge.net>.
 #
@@ -622,8 +622,8 @@
 # Identity of this package.
 PACKAGE_NAME='Net-SNMP'
 PACKAGE_TARNAME='net-snmp'
-PACKAGE_VERSION='5.9.4.pre2'
-PACKAGE_STRING='Net-SNMP 5.9.4.pre2'
+PACKAGE_VERSION='5.9.4'
+PACKAGE_STRING='Net-SNMP 5.9.4'
 PACKAGE_BUGREPORT='net-snmp-coders@lists.sourceforge.net'
 PACKAGE_URL=''
 
@@ -1673,7 +1673,7 @@
   # Omit some internal or obsolete options to make the list less imposing.
   # This message is too long to be a string in the A/UX 3.1 sh.
   cat <<_ACEOF
-\`configure' configures Net-SNMP 5.9.4.pre2 to adapt to many kinds of systems.
+\`configure' configures Net-SNMP 5.9.4 to adapt to many kinds of systems.
 
 Usage: $0 [OPTION]... [VAR=VALUE]...
 
@@ -1740,7 +1740,7 @@
 
 if test -n "$ac_init_help"; then
   case $ac_init_help in
-     short | recursive ) echo "Configuration of Net-SNMP 5.9.4.pre2:";;
+     short | recursive ) echo "Configuration of Net-SNMP 5.9.4:";;
    esac
   cat <<\_ACEOF
 
@@ -1859,7 +1859,7 @@
                                   libraries (eg, HPUX)
   --with-features-of=version
                           Compile in MIB modules and other features as if this
-                          was release "version" (default is 5.9.4.pre2).
+                          was release "version" (default is 5.9.4).
 
 Miscellaneous:
   --with-defaults         Use defaults for prompted values.
@@ -2184,7 +2184,7 @@
 test -n "$ac_init_help" && exit $ac_status
 if $ac_init_version; then
   cat <<\_ACEOF
-Net-SNMP configure 5.9.4.pre2
+Net-SNMP configure 5.9.4
 generated by GNU Autoconf 2.71
 
 Copyright (C) 2021 Free Software Foundation, Inc.
@@ -2978,7 +2978,7 @@
 This file contains any messages produced by compilers while
 running configure, to aid debugging if configure makes a mistake.
 
-It was created by Net-SNMP $as_me 5.9.4.pre2, which was
+It was created by Net-SNMP $as_me 5.9.4, which was
 generated by GNU Autoconf 2.71.  Invocation command line was
 
   $ $0$ac_configure_args_raw
@@ -36555,10 +36555,6 @@
 #       Check whether sysctl() is usable
 #	    used in library/agent
 #
-if test $cross_compiling = yes; then
-    { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: Can't check sysctl, manually define NETSNMP_CAN_USE_SYSCTL if platform support available" >&5
-printf "%s\n" "$as_me: WARNING: Can't check sysctl, manually define NETSNMP_CAN_USE_SYSCTL if platform support available" >&2;}
-else
     { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking if sysctl can read kernel information" >&5
 printf %s "checking if sysctl can read kernel information... " >&6; }
 if test ${ac_cv_NETSNMP_CAN_USE_SYSCTL+y}
@@ -36619,7 +36615,7 @@
 fi
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $ac_cv_NETSNMP_CAN_USE_SYSCTL" >&5
 printf "%s\n" "$ac_cv_NETSNMP_CAN_USE_SYSCTL" >&6; }
-fi
+
 
 if test "x$ac_cv_NETSNMP_CAN_USE_SYSCTL" = "xyes"; then
 
@@ -37648,7 +37644,7 @@
 # report actual input values of CONFIG_FILES etc. instead of their
 # values after options handling.
 ac_log="
-This file was extended by Net-SNMP $as_me 5.9.4.pre2, which was
+This file was extended by Net-SNMP $as_me 5.9.4, which was
 generated by GNU Autoconf 2.71.  Invocation command line was
 
   CONFIG_FILES    = $CONFIG_FILES
@@ -37716,7 +37712,7 @@
 cat >>$CONFIG_STATUS <<_ACEOF || ac_write_fail=1
 ac_cs_config='$ac_cs_config_escaped'
 ac_cs_version="\\
-Net-SNMP config.status 5.9.4.pre2
+Net-SNMP config.status 5.9.4
 configured by $0, generated by GNU Autoconf 2.71,
   with options \\"\$ac_cs_config\\"
 
